dashboard/README.md

```
@@ -1,23 +1,40 @@
 - [WorkOS AuthKit](https://workos.com/docs/authkit) - Authentication and user management
-- [MongoDB](https://www.mongodb.com/) - Database with Mongoose ODM
+- [QuestDB](https://questdb.io/) - High-performance time-series database for metrics
 - [tRPC](https://trpc.io) - Type-safe API layer
-- [Tailwind CSS](https://tailwindcss.com) - Styling
+- [Tailwind CSS 4](https://tailwindcss.com) - Styling
 - [shadcn/ui](https://ui.shadcn.com/) - UI components
 - [Recharts](https://recharts.org/) - Data visualization
 
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)